From 73b179877dd38aa58da43e83c83170f97dec20c0 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 27 Mar 2012 00:20:48 +0200
Subject: [PATCH 109/115] cat: fix priority type

Needs to be "int", not "char". Spotted by Frederic Crozat.
---
 src/journal/cat.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/journal/cat.c b/src/journal/cat.c
index 31d76f3..f0a6666 100644
--- a/src/journal/cat.c
+++ b/src/journal/cat.c
@@ -33,7 +33,7 @@
 #include "build.h"
 
 static char *arg_identifier = NULL;
-static char arg_priority = LOG_INFO;
+static int arg_priority = LOG_INFO;
 static bool arg_level_prefix = true;
 
 static int help(void) {
-- 
1.7.9.3

